packages/echo-core/src/services/hookRegistry/hookRegistry.ts

```diff
@@ -9,7 +9,9 @@ export enum RegisteredHookName {
     useSetActiveTagNo = 'useSetActiveTagNo',
     useContextMenuDataInfo = 'useContextMenuDataInfo',
     useTagData = 'useTagData',
-    useIsContextMenuInfoLoading = 'useIsContextMenuInfoLoading'
+    useIsContextMenuInfoLoading = 'useIsContextMenuInfoLoading',
+    useSetActiveCommPackNo = 'useSetActiveCommPackNo',
```

Comment:
Core should not know about business/domain logic, but only have general functionality. I think we should move this RegisteredHookName to framework. It shouldn't be this complicated to develop, that we have to change several repositories.

Reply:
Agree on this one: this enum is here because the hook registry is using it for incoming hook name.

I think we could simply just change the parameter type to plain string, and then we could move RegisteredHookName to framework.

Reply:
Yes, agree with this :)